### PR TITLE
Add Ashley to SSB

### PR DESCRIPTION
**Pull request by TheRun98**

### CODE REVIEW

**Comment by HoeenCoder** [comment]
@TheRun98 you should be doing all of these in 1 Pull Request with multiple commits in it.
Id suggest merging 2 of the branches into the one you want to keep and closing 2 of the PRs.

**Comment by HoeenCoder** [comment]
Waiting on ability before merging

File: mods/sgssb/abilities.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+exports.BattleAbilities = {
+
+// Ashley the Pikachu
+	primalsurge: {
+		shortDesc: "On switch-in, this Pokemon gets a 2x Speed boost on Electric Terrain.",
+		onStart: function (source) {
+			this.addPseudoWeather('electricterrain', source);
+		},
```
Comment (HoeenCoder) [inline mods/sgssb/abilities.js]: `isNonStandard: true,`

File: mods/sgssb/abilities.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+exports.BattleAbilities = {
+
+// Ashley the Pikachu
+	primalsurge: {
+		shortDesc: "On switch-in, this Pokemon gets a 2x Speed boost on Electric Terrain.",
+		onStart: function (source) {
+			this.addPseudoWeather('electricterrain', source);
```
Comment (HoeenCoder) [inline mods/sgssb/abilities.js]: Im not sure if this is possible but if it is this ability is supposed to work like primal weather, lasting as long as the user is active.

Comment (TheRun98) [reply mods/sgssb/abilities.js]: Yeah its weird. I don't really know how to work with that

**Comment by HoeenCoder** [comment]
Why was the abilities file removed?

**Pull request merged**